README.md

```diff
@@ -22,6 +22,8 @@ should be monitored by one exporter.
   - [`dbapi` (SAP/official)](https://help.sap.com/viewer/1efad1691c1f496b8b580064a6536c2d/Cloud/en-US/39eca89d94ca464ca52385ad50fc7dea.html)
   - [`pyhdb` (unofficial/open source)](https://github.com/SAP/PyHDB)
 
+3. For collecting much data as possible, you have to install the [SAP Host agent](https://launchpad.support.sap.com/#/notes/2101350).
```

Comment:
@juadk  Looks great. Maybe here we should rephrase a bit:
```suggestions
3. Some metrics are collected on the HANA monitoring views by the [SAP Host agent](https://launchpad.support.sap.com/#/notes/2101350). Make sure to have it installed and running to have access to all the monitoring metrics.
```

Reply:
Yes I agree with your suggestion, thanks.